test-db-upgrade/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+TMPDIR=$(mktemp -d)
+ENVDIR="$TMPDIR/env"
+
+# initialize the databases
+bash "$(dirname $0)"/init-db.sh
+
+# create virtualenv with jupyterhub 0.7
+echo "creating virutalenv in $ENVDIR"
+python -m virtualenv "$ENVDIR"
```

Comment:
Do you want `venv` instead?

Reply:
Good call. I started with venv and it wasn't working for some reason, so I installed `virtualenv` while I was testing and it fixed it 🤷‍♂️ . I'll probably switch back to venv once everything else is working.

Reply:
Turns out that I have to use virtualenv and not venv. The reason being that `python -m venv` doesn't work if run from an env created by `virtualenv`, which is the case on Travis (and happened to be the case on my local machine as well).